libraries/ms-common/inc/uart.h

```diff
@@ -43,3 +52,9 @@ StatusCode uart_set_rx_handler(UARTPort uart, UARTRxHandler rx_handler, void *co
 
 // Non-blocking TX
 StatusCode uart_tx(UARTPort uart, uint8_t *tx_data, size_t len);
+
+// Pops from TX Fifo and sends to serial device
+void prv_tx_pop(UARTPort uart);
```

Comment:
These are supposed to be private functions. Why are they exposed?

Reply:
That's my bad, I should have changed that back before pushing the code. In that case the interrupt handler should call prv_rx_push and prv_tx_pop.